snmp/datadog_checks/snmp/data/default_profiles/_generic-ups.yaml

```diff
@@ -73,23 +77,19 @@ metrics:
     symbol:
       OID: 1.3.6.1.2.1.33.1.6.1.0
       name: upsAlarmsPresent
-  - MIB: UPS-MIB
-    symbol:
-      OID: 1.3.6.1.2.1.33.1.7.3.0
-      name: upsTestResultsSummary
-      mapping:
-        1: donePass
-        2: doneWarning
-        3: doneError
-        4: aborted
-        5: inProgress
-        6: noTestsInitiated
-  # scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
-  # not currently supported
-  # - MIB: UPS-MIB
-  #   symbol:
-  #     OID: 1.3.6.1.2.1.33.1.7.4.0
-  #     name: upsTestResultsDetail
```

Comment:
note: Removing `upsTestResultsDetail` since the value is likely not suitable as tag.